tracking/translations_parser/parser.py

```diff
@@ -177,13 +178,23 @@ def _iter_log_entries(self) -> Iterator[tuple[list[tuple[str]], str]]:
                 self.run_date = self.get_timestamp(headers)
             text = line[position:]
 
+            def _join(iterable):
+                if not iterable:
+                    return "_"
+                if isinstance(iterable[0], str):
+                    return "_".join(iterable)
+                return _join([_join(item) for item in iterable])
+
             # Record logs depending on Marian headers
             if len(headers) >= 2:
                 # First is task timestamp, second is marian timestamp
                 _, _, *marian_tags = headers
-                tag = "_".join(*marian_tags) if marian_tags else "_"
+                tag = _join(marian_tags)
                 self.indexed_logs[tag].append(text)
 
+            # Display parsed log text on stderr for easier debug
+            print(f"Marian log: {text.strip()}", file=sys.stderr)
```

Comment:
Should this be removed? I think the idea was to just propagate the existing logging to stdout as.

Reply:
I replaced this by outputting the raw line unmodified, and prefixing log lines from parser with `[tracking]`